dataprofiler/profilers/numerical_column_stats.py

```diff
@@ -86,6 +86,7 @@ def __init__(self, options=None):
                     'bin_edges': None
                 }
             }
+        self._batch_history = []
```

Comment:
This could be changed to a queue of some fixed size if we don't want to store every single update history.